packages/core/src/Avatar/Avatar.styles.tsx

```diff
@@ -48,6 +48,17 @@ export const { staticClasses, useClasses } = createClasses("HvAvatar", {
     borderRadius: theme.radii.full,
     zIndex: 1,
   },
-  circular: { borderRadius: theme.radii.full },
+  circular: {
+    borderRadius: theme.radii.full,
+    " $avatar": {
+      borderRadius: theme.radii.full,
+    },
```

Comment:
can't we just `borderRadius: "inherit"` on `classes.avatar` instead? it's a direct child, and saves us of these duplicated overrides

Reply:
Yep, much cleaner.